### ISSUE TITLE
typo in comment in unix.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4979]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4979|
|Type|defect|
|Created|2011-03-22 21:28:50Z|

```
                        # do.  The bind() below will fail with an
                        # exception that actually propegates.
```

Should be _propagates_

Attachments:

* [unix-typo.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/8d8/8d828ec7ea9a3c6faf91f036e3f222c18e0dc987/024c0730b247371e8853db835c660f6b72679941.patch) (1066 bytes) - added by thijs on 2011-05-13 13:16:12Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4979 4979
type__defect defect
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1300829330000000 1300829330000000
changetime__1305495761000000 1305495761000000
version__None None
owner__thijs thijs
cc__thijs cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|ivank set owner to @thijstriemstra|
|-|-|

1. Please add an empty news file `twisted/topfiles/4979.misc`

Otherwise, looks good for merging.


(Out of curiosity, I just looked for spelling checkers that work on Python code, and found http://pypi.python.org/pypi/scspell .  Another thing I've done before is search for "it's" and "its"; I just found a misuse of "it's" in `postfix.py`.)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [ivank](#note_2):
> (Out of curiosity, I just looked for spelling checkers that work on Python code, and found http://pypi.python.org/pypi/scspell .  Another thing I've done before is search for "it's" and "its"; I just found a misuse of "it's" in `postfix.py`.)

That sounds like a useful tool that could be part of the documentation builder perhaps. If you find a lot of these errors, please open a new ticket, preferable with a patch :) Thanks, for the review, I'll merge this now.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [31906]) Apply unix-typo.patch: Correct a typo in the startListening docstring.

Author: thijs
Reviewer: ivank
Fixes: [#4979](https://github.com/twisted/twisted/issues/4979)
